tools/provision

```diff
@@ -48,37 +46,31 @@ the Python version this command is executed with."""
     venv_dir = os.path.join(base_dir, venv_name)
     if not os.path.isdir(venv_dir):
         try:
-            return_code = subprocess.call(['virtualenv', '-p', options.python_interpreter, venv_dir])
+            return_code = subprocess.call([options.python_interpreter, '-m', 'venv', venv_dir])
         except OSError:
-            if subprocess.call(['which', 'virtualenv']):
-                print("{red}Please install the virtualenv package and try again.{end_format}"
-                      .format(red='\033[91m', end_format='\033[0m'))
-                sys.exit(1)
+            print("{red}Installation with venv failed."
+                  "Maybe you are running an unsupported python version"
+                  "or python is not installed properly{end_format}"
+                  .format(red='\033[91m', end_format='\033[0m'))
+            sys.exit(1)
             raise
         else:
+            # subprocess.call returns 0 if a script executed successfully
             if return_code:
-                raise OSError("The command `virtualenv -p {} {}` failed. Virtualenv not created!"
+                raise OSError("The command `{} -m venv {}` failed. Virtualenv not created!"
                               .format(options.python_interpreter, venv_dir))
             print("New virtualenv created.")
     else:
         print("Virtualenv already exists.")
 
     if os.path.isdir(os.path.join(venv_dir, 'Scripts')):
         # POSIX compatibility layer and Linux environment emulation for Windows
-        # Virtual uses /Scripts instead of /bin on Windows.
-        # Read https://virtualenv.pypa.io/en/stable/userguide/
+        # venv uses /Scripts instead of /bin on Windows cmd and Power Shell.
+        # Read https://docs.python.org/3/library/venv.html
         venv_exec_dir = 'Scripts'
     else:
         venv_exec_dir = 'bin'
 
-    # In order to install all required packages for the venv, we need to activate it. Since
-    # the activation script sets environmental variables, it needs to be executed inline with
-    # `import_module`.
-    activate_module_dir = os.path.abspath(os.path.join(venv_dir, venv_exec_dir))
-    sys.path.append(activate_module_dir)
-
-    import_module('activate_this')
```

Comment:
Why is this no longer necessary?

Reply:
1. There is no activate_this in venv 
2. If you check the way packages are getting installed we don't actually need to activate the environment. To install packages we are fetching the path to pip in the venv and installing packages using that, which limits the package installation to the environment.